netbox/templates/extras/script.html

```diff
@@ -47,16 +47,44 @@
           {% csrf_token %}
           <div class="field-group my-4">
             {% if form.requires_input %}
-              <div class="row mb-2">
-                <h5 class="offset-sm-3">Script Data</h5>
-              </div>
+              {% if script.Meta.fieldsets %}
+                {# Render grouped fields according to declared fieldsets #}
+                {% for group, fields in script.Meta.fieldsets %}
+                  <div class="field-group mb-5">
+                    <div class="row mb-2">
+                      <h5 class="offset-sm-3">
+                        {% if group %}
+                          {{ group }}
+                        {% else %}
+                          {{ model|meta:"verbose_name"|bettertitle }}
```

Comment:
Group name should be required. This looks like it was copied inadvertently from elsewhere.

Reply:
yep